src/utils/validator_state.py

```diff
@@ -80,4 +80,7 @@ def calculate_total_active_effective_balance(validators: list[Validator], ref_ep
         if is_active_validator(v, ref_epoch):
             total_effective_balance += int(v.validator.effective_balance)
 
+    if total_effective_balance == 0:
```

Comment:
This is wrong

Reply:
Unfortunately, it's possible here.

Stack:
1. MidtermSlashingPenalty.get_validator_midterm_penalty::172
2. MidtermSlashingPenalty.predict_midterm_penalty_in_frame
3. frames_lido_midterm_penalties = MidtermSlashingPenalty.get_future_midterm_penalty_sum_in_frames(
            blockstamp.ref_epoch, all_slashed_validators,  total_balance, future_frames_lido_validators,
 )
4. total_balance = src/services/bunker_cases/midterm_slashing_penalty.py::58

![Screenshot 2023-03-20 at 13 28 22](https://user-images.githubusercontent.com/102730938/226299698-039f2503-ff97-4a8e-b7ca-4e21d813b8c7.png)